main/fastcgi.c

```diff
@@ -1209,6 +1223,12 @@ static int fcgi_read_request(fcgi_request *req)
 
 	return 1;
 }
+#else // __wasi__
+static int fcgi_read_request(fcgi_request *req)
```

Comment:
Updated the patch; `fastcgi.c` requires some changes because `cgi_main` uses certain symbols that come from `fastcgi.c`; and we are interested in `php-cgi`.